SAMPLE_FORMAT.md

```diff
@@ -199,6 +202,16 @@ Snippets should have tests that should verify the snippet works and compiles cor
 The following are some general Java best practices that should be followed in samples to remain
 idiomatic. 
 
+### Modern Java
```

Comment:
In order to make these easier to reference, can we separate them into different sections? It sounds like there are three sections:

- Labdas
- Streams
- For-each loops over `ArrayLists.forEach()`